text/0000-buildpackless-builders.md

```diff
@@ -0,0 +1,330 @@
+project file
+(`project.toml`)](https://buildpacks.io/docs/reference/config/project-descriptor/),
+users can specify the buildpack(s) for their build in an easily stored and
+reproducible way. Users can, of course, also specify which buildpacks to
+include in the build with the `--buildpack` flag for pack.
+
+The proposed new builders will thus speed up users' builds and Paketo's
+buildpack tests while relying only upon the existing functionality that the
+pack CLI provides.(No upstream contributions required.)
+
+
+## Detailed Explanation
+
+### Definitions
+- **buildpackless builder** : A
+  [builder](https://buildpacks.io/docs/concepts/components/builder/) that
+  includes a build-image, a reference to a run-image, and a lifecycle, _but no
+  buildpacks nor order groups_.
+- **standard builder**: A builder that includes buildpacks and order groups in
+  addition to the build-image, lifecycle, and reference to a run-image.
+
+### Calculating Builder Size
+
+To determine the number of layers in a given builder image, one can use the
+`docker history` command. Its output shows one line per layer. For example, to
+calculate the number of layers in the latest Paketo Full builder, simply run:
+```
+docker pull paketobuildpacks/builder:full
+docker history -q paketobuildpacks/builder:full | wc -l
+```
+
+### What will the Paketo Buildpackless Builders look like?
+The `buildpackless-builder.toml` for a given Paketo Buildpackless <Name> Builder will be identical
+to the corresponding standard Paketo <Name> Builder, but will have the `[[
+buildpacks ]]` and `[[ order ]]` sections removed. For instance, the Buildpackless
+Full Builder will have a `buildpackless-builder.toml` that looks like:
+
+```
+description = "<some description that clearly states buildpacks must be specified with this builder>"
+
+[lifecycle]
+  version = "X.Y.Z"
+
+[stack]
+  build-image = "docker.io/paketobuildpacks/build:X.Y.Z-full-cnb"
+  id = "io.buildpacks.stacks.bionic"
+  run-image = "index.docker.io/paketobuildpacks/run:full-cnb"
+  run-image-mirrors = ["gcr.io/paketo-buildpacks/run:full-cnb"]
+```
+
+The above stated change results in a ~88% decrease in the number of layers
+as compared to Paketo Full Builder v0.1.130.
+
+### How will the Paketo Buildpackless Builders be distributed?
+Paketo will maintain and publish buildpackless builders that correspond to each
+of its three existing standard builders. Their config files
+(`buildpackless-builder.toml`) will be checked in to the same repos as the
+standard builders (paketo-buildpacks/full-builder, etc.).  They'll be pushed to
+Dockerhub and GCR using tags `paketobuildpacks/builder:buildpackless-full` for
+the latest builder and `paketobuildpacks/builder:1.2.3-buildpackless-full` for
+version 1.2.3 of the builder. This parallels the [current tagging
+syntax](https://hub.docker.com/r/paketobuildpacks/builder/tags?page=1&ordering=last_updated)
+of `paketobuildpacks/builder:full` and `paketobuildpacks/builder:1.2.3-full`.
```

Comment:
On first read I thought this conflicted with the plan to version them separately (the paragraph below) but I realize `paketobuildpacks/builder:1.2.3-buildpackless-full` is referrring the buildpackless builder version not the full builder version. Maybe we can update this to make it less ambigous? I misinterpreted which builder 
> `paketobuildpacks/builder:1.2.3-buildpackless-full` for
version 1.2.3 of the builder.

 was referring to.